nautobot/docs/installation/http-server.md

```diff
@@ -138,16 +138,47 @@ Finally, restart the `nginx` service to use the new configuration.
 $ sudo systemctl restart nginx
 ```
 
+!!! info
+    If the restart fails, and you changed the default key location, check to make sure the `nautobot.conf` file you pasted has the updated key location. For example, CentOS 8.2.2 requires keys to be in `/etc/pki/tls/` instead of `/etc/ssl/`.
```

Comment:
Is `8.2.2` here versus `8.2.2004` above an intentional distinction?

Reply:
oops.  no.  fixed it